receiver/prometheusreceiver/metrics_receiver.go

```diff
@@ -138,6 +139,7 @@ func (pr *Preceiver) StartMetricsReception(host receiver.Host) error {
 		ctx := host.Context()
 		c, cancel := context.WithCancel(ctx)
 		pr.cancel = cancel
+		c = observability.ContextWithReceiverName(c, pr.MetricsSource())
```

Comment:
For the tag value you should probably use the observability.ComponentName that includes also the type + name from the component config.

Reply:
@bogdandrutu - done, ptal